website-scraper/index.d.ts

```diff
@@ -0,0 +1,48 @@
+// Type definitions for website-scraper v1.2.x
+// Project: https://github.com/s0ph1e/node-website-scraper
+// Definitions by: Christian Rackerseder <https://www.echooff.de>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "website-scraper" {
```

Comment:
yes. like this https://github.com/DefinitelyTyped/DefinitelyTyped/pull/12038/commits/e52e881a914360fd5f06245adb7522931d336e80?w=